mobify/platform-scaffold
# Sets up template for error page from lancome build.

Sets up template for error page from lancome build.

 **JIRA**: https://mobify.atlassian.net/browse/AMP-81
**Status**: Ready for Review

## Changes
- Adds error container and styles, i.e. 
<img width="300" alt="screen shot 2017-09-19 at 2 07 35 pm" src="https://user-images.githubusercontent.com/16738243/30615480-f98bacfe-9d43-11e7-920e-455799798735.png">

- Adds analytics for error page
- Incorporates canonicalURL utility function to anywhere that uses a custom URL building function
- Bumps `mobify-amp-sdk` version to 0.7.2

## Todos
- [x] make it work

## How to test-drive this PR
- `cd amp && npm run dev`
- visit `localhost:3000/randomroute` to see the static error page
- visit `localhost:3000/potions.html`  || `http://localhost:3000/potions/eye-of-newt.html`
to ensure PLP/PDP works
